common/errors.js

```diff
@@ -142,12 +142,16 @@
                 window.location.reload();
             }; 
             if (exceptionFlag || window.location.pathname.indexOf('/search/') != -1 || window.location.pathname.indexOf('/viewer/') != -1) return;
-
+            
             if (ERMrest && exception instanceof ERMrest.UnauthorizedError || exception.code == errorNames.unauthorized) {
                 Session.loginInAModal(reloadCb);
             }
             else if (exception.code && exception.code == errorNames.multipleRecords){
                 errorPopup(messageMap.multipleDataMessage, messageMap.multipleDataErrorCode,"Recordset ", exception.redirectUrl);
+            }
+            
+            else if (ERMrest && exception instanceof ERMrest.UnauthorizedError || exception.code == errorNames.forbidden) {
```

Comment:
Can you change this to have some parenthesis to group the logic statements together? Right now it evaluates from left to right which is probably correct, but it's easier to understand with parenthetic grouping.

Reply:
I think it's supposed to be read as:
```
( (ERMrest && exception instanceof ERMrest.UnauthorizedError) || exception.code == errorNames.forbidden )
```